### ISSUE TITLE
Shut Down Distributed TPU Error

### ISSUE THREAD

**Issue by brijraj08** [tensorflow/models#4055]
I'm trying to run the given code of MobileNet V2 for checkpoint based inference, shared on Colab  with given pre-trained model (mobilenet_v2_0.75_160.ckpt.data-00000-of-00001).
When I run this code on Colab or on my PC, I find the error as: 
 [[Node: ShutdownDistributedTPU = ShutdownDistributedTPU[_device="/job:tpu_worker/device:TPU_SYSTEM:0"]()]]
Please look into the problem.

**Comment by nealwu** [comment]
@marksandler2 could you take a look?

**Comment by marksandler2** [comment]
@brijraj08 
Can you share a code snippet, the supported way to restore from a checkpoint is shown here:

https://colab.sandbox.google.com/github/tensorflow/models/blob/master/research/slim/nets/mobilenet/mobilenet_example.ipynb

basically:

```
logits, endpoints = mobilenet_v2.mobilenet(images, depth_multiplier=0.75)
ema = tf.train.ExponentialMovingAverage(0.999)
vars = ema.variables_to_restore()
saver = tf.train.Saver(vars)  
with tf.Session() as s:
  saver.restore()
```
You shouldn't try to restore from metagraph, because it then tries to replicate a full training setup, which is probalby not what you want (because the graph would be in training mode and expect certain hardware setup)

Then if you want you can re-dump the meta graph

**Comment by brijraj08** [comment]
okay @marksandler2 , I got the mistake, thanks for your reply.

**Comment by wt-huang** [comment]
Closing as this is resolved
